examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>Recommend: to pip install -r PySyft/requirements.txt</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>To have an actual executable command, I added:</p><p><br></p><p><code>pip install -r "../../../requirements.txt"</code></p>